Source/Editor/DualityEditor/DualityEditor.csproj

```diff
@@ -6,6 +6,8 @@
     <StartWorkingDirectory>$(SolutionDir)Build\Output\</StartWorkingDirectory>
   </PropertyGroup>
   <PropertyGroup>
+    <RestoreProjectStyle>PackageReference</RestoreProjectStyle>
```

Comment:
Looks like the editor project attempts to resolve packages via PackageReference, but the `csproj` itself doesn't contain any and instead relies on `packages.config` doing its job. Could be the inconsistency that causes most of the build errors in the latest CI build?